modules/gitops-storing-and-retrieving-argo-cd-logs.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * cicd/gitops/viewing-argo-cd-logs-using-the-kibana-dashboard.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-storing-and-retrieving-argo-cd-logs_{context}"]
+= Storing and retrieving Argo CD logs
+
+You can use the Kibana dashboard to store and retrieve Argo CD logs.
+
+.Procedure
+
+. In the {product-title} web console, navigate to the {rh-app-icon} menu -> *Observability* -> *Logging*. The Kibana dashboard is opened in a new window.
+
+. Create an index pattern.
+
+.. To display all the indeces, define the index pattern as "***" . Click *Next step* to continue.
+
+.. Select *@timestamp* for _Time Filter field name_.
+
+.. Click *Create index pattern* and wait until the index pattern is created.
+
+. In the left navigation panel of the Kibana dashboard, click the *Discover* tab.
+
+. Create a filter to retrieve logs for Argo CD.
+
+.. Click *Add a filter +*.
+
+.. Provide `kubernetes.namespace_name` as the field. Select the "*is*" operator and enter `openshift-gitops` as a value.
```

Comment:
Seems like this list items contains three different steps.

Reply:
The place where the user enters `kubernetes.namespace_name` is specifically named "Fields", that is why I am calling it "field". However, I will rephrase it as you suggested for better clarity, thanks!

Reply:
- These are dropdown lists and not user-entered text, so use "select" rather than "provide". 
- Also UI elements, including list items in dropdown lists, should be in bold type.

- Maybe consider creating 3 short steps instead of a single step:

```
.. Select the *kubernetes.namespace_name* namespace. 
.. Select the *is* operator.
.. Select the *openshift-gitops* group.
```

- Also, even though the **Filter** dropdown lists do not have labels, you should add descriptive words as in my example. (I don't know if "namespace" and "group" are the correct names, btw, so you should double-check that.)